english/swiper/dora.md

```diff
@@ -0,0 +1,4 @@
+You see two tunnels . Before them is Dora asking you the way to find the marshmellow palace .She asks you which tunnel to take that has no swiper waiting to swipe the marshmellows.oh man!
+Do you go [left?](goleft/goleft.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.